spec/features/audit/report/csv_export_spec.rb

```diff
@@ -13,99 +7,140 @@ def content_disposition
+    given_i_am_an_auditor_belonging_to_an_organisation
+    and_there_are_multiple_pages_of_unfiltered_content_items
+    when_i_export_all_the_reports
+    then_i_receive_an_audit_progress_report_for_all_content_items_in_csv_format
+  end
 
-      expect(page).to have_content("Title,URL")
-      expect(page).to have_content("Example 1,https://gov.uk/example1")
-    end
+  scenario "Discard audit status filter when clicking from content view to report, and then exporting CSV" do
+    given_i_am_an_auditor_belonging_to_an_organisation
+    and_there_are_two_unassigned_content_items
+    when_i_filter_the_content_page_for_content_items_with_no_one_assigned_to_them
+    then_i_see_the_two_unassigned_content_items
+    when_i_navigate_to_audits_report_page
+    then_i_do_not_see_the_two_unassigned_content_items
+    and_i_see_the_one_content_item_assigned_to_me
+  end
 
-    scenario "Applying the filters to the export" do
-      visit audits_report_path
-      select "Anyone", from: "allocated_to"
+  def given_i_am_an_auditor_belonging_to_an_organisation
+    user = create(:user)
+    hmrc = create(
+      :content_item,
+      title: "HMRC",
+      document_type: "organisation",
+      allocated_to: user
+    )
+    example1 = create(:content_item,
+                      title: "Example 1",
+                      base_path: "/example1",
+                      allocated_to: user)
+    create(:link,
+           source_content_id: example1.content_id,
+           target_content_id: hmrc.content_id,
+           link_type: "primary_publishing_organisation")
+    create(:audit, content_item: example1)
+  end
 
-      select "HMRC", from: "Organisations"
-      click_on "Apply filters"
+  def then_i_receive_only_hmrc_related_content_items_in_the_report
+    expect(@audit_report).to have_content("Title,URL")
+    expect(@audit_report).to have_content("Example 1,https://gov.uk/example1")
+    expect(@audit_report).to have_no_content("Example 2,https://gov.uk/example2")
+  end
 
-      click_link "Export filtered audit to CSV"
-      expect(page).to have_content("Example 1,https://gov.uk/example1")
-      expect(page).to have_no_content("Example 2,https://gov.uk/example2")
-    end
+  def then_i_see_the_two_unassigned_content_items
+    expect(@audit_content_page).to have_no_content("Example 1")
+    expect(@audit_content_page).to have_content("Example 2")
+    expect(@audit_content_page).to have_content("Example 3")
+  end
 
-    scenario "Discard audit status filter when clicking from content view to report, and then exporting CSV" do
-      visit audits_path
-      select "Anyone", from: "allocated_to"
+  def when_i_navigate_to_audits_report_page
+    @audit_content_page.audits_progress_tab.click
+  end
 
-      choose "Audited"
+  def then_i_do_not_see_the_two_unassigned_content_items
+    expect(@audit_report_page).to be_displayed
+    expect(@audit_report_page).to have_no_content('2 Content items')
```

Comment:
This is a dangerous assertion to rely on. This test could incorrectly pass if:
- we have 0 content items
- we have 1 content item
- we have 3 content items
- (you see where I'm going)
- and _also_ if we **do** have 2 content items, but we've reworded the copy to read eg "2 items"

Reply:
As discussed yesterday, this test is asserting that our current filter is not persisted when we navigate to a new page. I test that we have exactly `1 content item` on the next line but broke the two assertions down into individual method names to try to help a slightly weird test to be more easily understood.